TeamPage.html

```diff
@@ -419,6 +419,56 @@ <h2>Meet Our Team</h2><br>
             </div>
           </div>
         </div>
+        <div class="col-sm-3 col-xs" style="min-width: 210px;">
+          <div class="id_presentation">
+            <img class="img_staff"
+              src="https://avatars.githubusercontent.com/u/153774845?v=4"
+              style="width: 200px; height: 200px; object-fit: cover; object-position: top;" alt="Zach's Photo" />

```

Comment:
Again, super minor thing but can the alt-text be "Zach's Picture" instead of "Photo" to match all the other profiles? Thank you!

Reply:
Great catches, I believe i remedied each issue.